dmt/api/auth.py

```diff
@@ -49,7 +49,7 @@ def _has_safe_remote_host(self, remote_host):
             return False
 
     def has_permission(self, request, view):
-        remote_addr = request.META.get('REMOTE_ADDR')
+        remote_addr = request.META.get('HTTP_X_REAL_IP')
```

Comment:
I'm thinking we probably want something like `request.META.get('HTTP_X_REAL_IP', request.META.get('REMOTE_ADDR'))` so it would also work if it it's deployed without a proxy.